src/web/layouts/StandardLayout.tsx

```diff
@@ -336,22 +336,26 @@ export const StandardLayout = ({
     const { branding } = CAPI.commercialProperties[CAPI.editionId];
     return (
         <>
-            <div>
+            <div data-print-layout='hide'>
```

Comment:
do you need to repeat `data-print-layout='hide'` on child DOM nodes?

Reply:
Where does standard layout get that attribute from?

Reply:
Where would you recommend putting that attribute?

Reply:
I tried changing the attribute up the hierarchy and it failed to work. Unless there are other methods I'm happy that this works for now.